### DISCUSSION TITLE
Problems with WipRpinterface

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#1475] [Q&A] [answered]
hi,I'm using the Imodelbank service, I'm connecting to multiple versions of Imodel, and then I'm using the API in the Wiprp Interface,

These APIs are all suspended.The program is no longer executing forward.

I would like to ask, are there any restrictions on the use of these RPC APIs in the RPC WIPRpInterface?Thanks.

**Comment by pmconne** [comment, marked as answer]
As stated in its documentation, WipRpcInterface is for internal development purposes only and never intended for general use.
